src/js/vault-client/vault-api.js

```diff
@@ -58,12 +59,18 @@ const fetchVaultToken = async () => {
 
 export const initVaultPwd = (idep) => {
 	axiosVault(`${VAULT_BASE_URI}/v1/${VAULT_KV_ENGINE}/data/${idep}/.onyxia/profile`)
-		.then(({ data: { data : {data}} }) => store.dispatch(newVaultData(data)))
+		.then(({ data : {data} }) => 
+			pwdMustBeRenewed(data.metadata.created_time) ? resetVaultPwd(idep) : store.dispatch(newVaultData(data.data))
+			)
 		.catch(() => {
 			resetVaultPwd(idep);
 		});
 };
 
+const pwdMustBeRenewed = (created_time) => {
+	return Date.now() - Date.parse(created_time) > 8400*NB_DAYS_BEFORE_PWD_RENEWAL
```

Comment:
I may be missing something here, what is `8400` equivalent to ? Is is supposed to `86400` (number of seconds in a day) ?